doc-source/xray-sdk-dotnet-messagehandler.md

```diff
@@ -50,23 +50,41 @@ namespace SampleEBWebApplication
 
 ## Instrumenting incoming requests \(\.NET Core\)<a name="xray-sdk-dotnet-messagehandler-startupcs"></a>
 
-To instrument requests served by your application, call the `UseExceptionHandler`, `UseXRay`, and `UseStaticFiles` methods in the `Configure` method of your `Startup` class\.
+To instrument requests served by your application, call `UseXRay` method before any other middleware in the `Configure` method of your `Startup` class\.
+
+**Note**
+
+For .NET Core 2.0, if you have `UseExceptionHandler` method in the application, make sure to call `UseXRay` after `UseExceptionHandler` method so as to record exceptions\.
 
 **Example Startup\.cs**  
 
+****.NET Core 2.1 and above****
+
 ```
 using Microsoft.AspNetCore.Builder;
 
 public void Configure(IApplicationBuilder app, IHostingEnvironment env)
   {
-    app.UseExceptionHandler("/Error");
     app.UseXRay("MyApp");
+    app.UseExceptionHandler("/Error");
     app.UseStaticFiles();
```

Comment:
We can remove the irrelevant middlewares like `UseStaticFiles` and `UseMVC` from the examples as well.

Reply:
If so, we may need to update the doc [here](https://github.com/aws/aws-xray-sdk-dotnet#aspnet-core-framework-net-core--nuget) too.